cypress/e2e/read-receipts/read-receipts-utils.ts

```diff
@@ -0,0 +1,442 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import type { MatrixClient, MatrixEvent, Room, IndexedDBStore } from "matrix-js-sdk/src/matrix";
+import Chainable = Cypress.Chainable;
+
+export class MessageFinder {
```

Comment:
I'm much of the opinion that once things start being `export`ed and shared between multiple files, they really ought to have a doc-comment. Apart from anything else, having a written mission statement for a class helps guard against feature creep where you end up with a grab-back of things that don't really relate to each other.

But, that's probably a job for another day.

Reply:
Thank you - a job for today: bf146eddd1